examples/sensor-websocket/src/components/SensorData.jsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { Alert, Col, Container, ListGroup, Row } from "react-bootstrap";
+
+function ListItem(props) {
+  return (
+    <ListGroup.Item
+      key={props.index}
+      className="temperature-list px-3 border-0 pb-0 py-2 bg-transparent"
+    >
+      <b>{props.label}: </b>
+      <span>{props.value}</span>
+    </ListGroup.Item>
+  );
+}
+
+const setSensorDataCard = (index, sensor) => {
+  const sensor_item = [
+    {
+      label: "Value",
+      value: sensor.value
+    },
+    {
+      label: "Last Update",
+      value: sensor.timestamp
+    }
+  ];
+  return (
+    <Row key={index} className="m-3 bg-light px-2 pt-3 pb-2">
+      <Col className="text-dark px-3 py-2 bg-transparent border-0 font-weight-bold w-100 text-left btn text-decoration-none">
+        <span className="bg-sensor-theme font-weight-light py-2 px-3 w-100 text-white d-inline-block">
+          <p className="font-weight-bold">Sensor Name : {sensor.name}</p>
+        </span>
+        <Col className="p-0">
```

Comment:
here you have a Col inside an other Col, that shouldn't happen when the layout classes are used properly.

Reply:
I need to test properly with data but right now no device streaming

Reply:
Done